sympy/concrete/summations.py

```diff
@@ -518,6 +519,12 @@ def is_convergent(self):
                 except NotImplementedError:
                     pass
 
+        ### ------------- Limit comparison test -----------###
+        # (n) comparison
+        lim_comp = limit((sequence_term/sym), sym, oo)
+        if lim_comp is not S.Infinity:
```

Comment:
If `lim_comp` is `oo`, then also `limit(sequence_term, sym, oo)` must be `oo`.  Therefore this test will return `False` for all sequences whose terms have finite limit.
```
>>> Sum(sin(n)/n, (n, 1, oo)).is_convergent()
False
```

Reply:
Yes. 
@jksuom , It was a very abrupt implementation. I will create a more robust one.

Reply:
The implementation has been tweaked. Please see [here](https://math.oregonstate.edu/home/programs/undergrad/CalculusQuestStudyGuides/SandS/SeriesTests/limit_comparison.html) for the theory part. Only the divergence comparison, with the `harmonic series` has been implemented.
Now --

```
>>> Sum(sin(n)/n, (n, 1, oo)).is_convergent()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "sympy/concrete/summations.py", line 555, in is_convergent
    "is not yet implemented" % (sequence_term))
NotImplementedError: The algorithm to find the Sum convergence of sin(n)/n is not yet implemented
```
The description has also been modified.